proposals/NNNN-keep-rc-app-hmi-level-when-disable-rc.md

```diff
@@ -0,0 +1,48 @@
+# SDL shall not put RC applications to HMI level NONE when user disables RC in HMI
+
+* Proposal: [SDL-NNNN](NNNN-keep-rc-app-hmi-level-when-disable-rc.md)
+* Author: [Zhimin Yang](https://github.com/smartdevicelink/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [ Core / Web]
```

Comment:
Can you please advise which Web component changes are required for this proposal?

Reply:
It shall be sdl hmi, not web.